### ISSUE TITLE
How to build a patch version

### ISSUE THREAD

**Issue by rfay** [crazywhalecc/static-php-cli#190]
I'd like to figure out how to build a patch version, like php 7.4.32 maybe.

I guess I figured it out, with:
```
bin/spc download --all -U "php-src:https://www.php.net/distributions/php-7.4.32.tar.gz"
./bin/spc build bcmath,openssl,tokenizer,sqlite3,pdo_sqlite,ftp,curl --build-cli --build-fpm
./buildroot/bin/php --version
PHP 7.4.32 (cli) (built: Sep 15 2023 11:18:05) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
```

But the canonical way to do this would improve the README. It looks like the key is deleting the `downloads` and `source` directories first? The first time I did it I got 7.4.33, which I had apparently been built earlier.

Actually supporting the patch version in `spc fetch` would be a win too.

My goal is to provide a workaround for the occasional [DDEV](https://github.com/ddev/ddev) user who wants a particular patch version for testing or comparison. It's certainly not common.

**Comment by stloyd** [comment]
There is a `--clean` option for download which will delete previously downloaded files & extracted source.

**Comment by crazywhalecc** [comment]
If you just want to compile a different version of PHP immediately after compiling one version of PHP, you can consider deleting the `source` directory and `downloads/php-*.tar .gz` file, and then run the `bin/spc download` command and the `bin/spc build` command again.

**Comment by rfay** [comment]
@stloyd it does seem that `--clean` is incompatible with all the other arguments; if you use it nothing else happens. For example, with `bin/spc download --clean --all  -U "php-src:https://www.php.net/distributions/php-${STATIC_PHP_VERSION}.tar.gz"`

**Comment by crazywhalecc** [comment]
@rfay Yeah. It's just an alias to `rm -rf` those dirs. I'm not sure people need a `bin/spc clean:download` command.

**Comment by rfay** [comment]
Thanks for the help, all.
